classes/codebench.bbclass

```diff
@@ -41,6 +53,20 @@ python codebench_check () {
     else:
         d.setVar('EXTERNAL_TARGET_SYS', triplets[0])
         d.setVar('EXTERNAL_TOOLCHAIN', os.path.join(toolchains_path, toolchain_subdir))
+
+python codebench_check () {
+    codebench_path = d.getVar('CODEBENCH_PATH', True)
+    exttc = d.getVar('EXTERNAL_TOOLCHAIN', True)
+    if exttc:
+        if codebench_path:
+            # Explicitly set EXTERNAL_TOOLCHAIN is preferred
```

Comment:
When both paths are set I think we should print a Note that Toolchain Path is preferred and we are picking this patch